forta-scanner-node-software-updated/src/agent.js

```diff
@@ -7,18 +7,21 @@ const handleTransaction = async (txEvent) => {
   const txFiltered = txEvent.filterLog(abi, contract);
 
   txFiltered.forEach((tx) => {
+    const { from } = tx;
     const { oldVersion, newVersion } = tx.args;
-
+    const fromLowerCase = from.toLowerCase();
```

Comment:
I got "Cannot read property 'toLowerCase' of undefined" when I tried the test transaction from the README. I think the problem is that the event doesn't have "from" property. Maybe you meant "address"?

Reply:
Also the event.address property should already have only lower case letters